lib/Remotery.c

```diff
@@ -4369,6 +4382,14 @@ static void SetDebuggerThreadName(const char* name)
     #else
         RMT_UNREFERENCED_PARAMETER(name);
     #endif
+
+    #ifdef RMT_PLATFORM_POSIX
+        // pthread_setname_np is a non-standard GNU extension.
+        char name_clamp[16];
+        strncpy(name_clamp, name, 15);
```

Comment:
strncpy is unsafe here. Can you set `name_clamp[0]=0` and call `strncat_s` instead?